OP-10984 - Update IRONdb Config Information (#57)
circonus/product-docs: irondb/getting-started/configuration.md

Default: "raw\_iterator"

#### raw_database sync_after_full_rollup_finishes

> Enables doing a manual LMDB sync to disk after a raw shard finishes rolling up.

Reviewer: ```suggestion
Enables an LMDB sync to disk after each raw shard finishes rolling up.
```

Each shard that the raw shard rolls up into will be synced.

Default: "false"

#### raw_database sync_after_column_family_rollup_finishes

> Enables doing a manual LMDB sync to disk after each column family within a raw shard

Reviewer: ```suggestion
Enables an LMDB sync to disk after each column family within a raw shard
```
